x-pack/plugins/lists/common/schemas/request/update_exception_list_item_schema.ts

```diff
@@ -13,13 +13,19 @@ import {
   Tags,
   _Tags,
   _tags,
+  created_at,
```

Comment:
I think the reason these weren't there is because we don't want to allow the user to send these items up since they are items that are set by the server.

I might defer to @FrankHassanabad , but I think we should leave these out.

Reply:
Done. Reverted this change.